ddev/ddev
# fix: hide obsolete ddev config --http-port

## The Issue

In testing 
* https://github.com/ddev/ddev/pull/6002

I happened to notice that `--http-port` and `--https-port` are still shown in `ddev config -h`. But we don't really want those to be used any more.

## How This PR Solves The Issue

Hide them

## Manual Testing Instructions

* `ddev config -h` should not show those
* `ddev config --http-port=8080 --https-port=8443` should still work and add that config to .ddev/config.yaml

Don't forget to remove the config from your .ddev/config.yaml

## Automated Testing Overview



## Related Issue Link(s)

## Release/Deployment Notes
